e2e_samples/fabric_dataops_sample/infrastructure/terraform/main.tf

```diff
@@ -115,19 +114,73 @@ module "fabric_spark_custom_pool" {
   custom_pool_name = local.fabric_custom_pool_name
 }
 
-module "fabric_notebook" {
+module "fabric_setup_notebook" {
   source                   = "./modules/fabric/notebook"
   workspace_id             = module.fabric_workspace.workspace_id
-  notebook_name            = local.fabric_notebook_name
-  notebook_definition_path = local.notebook_definition_path
+  notebook_name            = local.fabric_setup_notebook_name
+  notebook_definition_path = local.setup_notebook_definition_path
   tokens = {
-    "workspace_name" = module.fabric_workspace.workspace_name
     "lakehouse_name" = module.fabric_lakehouse.lakehouse_name
+    "lakehouse_id"   = module.fabric_lakehouse.lakehouse_id
+    "workspace_id"   = module.fabric_workspace.workspace_id
+  }
+}
+
+module "fabric_standardize_notebook" {
+  source                   = "./modules/fabric/notebook"
+  workspace_id             = module.fabric_workspace.workspace_id
+  notebook_name            = local.fabric_standardize_notebook_name
+  notebook_definition_path = local.standardize_notebook_definition_path
+  tokens = {
+    "lakehouse_name" = module.fabric_lakehouse.lakehouse_name
+    "lakehouse_id"   = module.fabric_lakehouse.lakehouse_id
+    "workspace_id"   = module.fabric_workspace.workspace_id
+  }
+}
+
+module "fabric_transform_notebook" {
+  source                   = "./modules/fabric/notebook"
+  workspace_id             = module.fabric_workspace.workspace_id
+  notebook_name            = local.fabric_transform_notebook_name
+  notebook_definition_path = local.transform_notebook_definition_path
+  tokens = {
+    "lakehouse_name" = module.fabric_lakehouse.lakehouse_name
+    "lakehouse_id"   = module.fabric_lakehouse.lakehouse_id
+    "workspace_id"   = module.fabric_workspace.workspace_id
+  }
+}
+
+# The service principal/managed identity is not grated RBAC permissions to access secrets.
+# Therefore, the secrets are stored in the AKV only when using user context (var_use_cli==true).
+module "key_vault_secret_001" {
+  enable       = var.use_cli
+  source       = "./modules/keyvault_secret"
+  name         = var.kv_appinsights_connection_string_name
+  value        = module.appinsights.connection_string
+  key_vault_id = module.keyvault.keyvault_id
+  content_type = "Application Insights Connection String"
+  tags         = local.tags
+}
+
+# Below modules currently do not support service principal/managed identity execution context.
+# Therefore they are enabled only when using user context (var_use_cli==true).
```

Comment:
This is currently not referred in the readme - it needs to be.

Reply:
This has been added in the [feat/e2e-fabric-dataops-sample](https://github.com/Azure-Samples/modern-data-warehouse-dataops/tree/feat/e2e-fabric-dataops-sample) branch.